src/tasks/LibraryBuilder/Templates/CMakeLists.txt.template

```diff
@@ -5,21 +5,20 @@ project(%LIBRARY_NAME%)
 enable_language(%CMAKE_LANGS%)
 
 set(DOTNET_AOT_SOURCES
-    %AotSources%
+%AotSources%
 )
 set(DOTNET_AOT_OBJECTS
-    %AotObjects%
+%AotObjects%
 )
 set(DOTNET_EXTRA_SOURCES
-    %ExtraSources%
-    assembly_list.c
+%ExtraSources%    preloaded_assemblies.c
```

Comment:
put them on separate lines to increase readability.

Reply:
Initially I placed it as such so the resulting CMakeFile after replacing `%ExtraSources%` wouldn't have weird spaces or extra empty lines.

I'll rearrange it as

```
set(DOTNET_EXTRA_SOURCES
    preloaded_assemblies.c
%ExtraSources%)
```

so if `%ExtraSources%` is empty
```
set(DOTNET_EXTRA_SOURCES
    preloaded_assemblies.c
)
```
and otherwise
```

set(DOTNET_EXTRA_SOURCES
    preloaded_assemblies.c
    extrasource_1
    extrasource_2
    ...
    extrasource_n
)
```